pkg/csi/service/common/commonco/k8sorchestrator/topology.go

```diff
@@ -1558,8 +1565,62 @@ func (volTopology *wcpControllerVolumeTopology) GetTopologyInfoFromNodes(ctx con
 
 	switch strings.ToLower(params.StorageTopologyType) {
 	case "zonal":
-		// If the topology requirement received has just one zone, use the same zone as node affinity terms on PV.
-		if len(params.TopologyRequirement.GetPreferred()) == 1 {
+		if params.TopologyRequirement == nil {
+			// If the topology requirement received is nil, then identigy topology of the datastore by looking into
```

Comment:
Can we add a comment saying it is for static provisioning case

Reply:
typo: identigy -> identify

Reply:
done